src/styles/main.scss

```
@@ -12,3 +12,22 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+.catalog__card {
+  transition: all 300ms;
```

```suggestion
  transition: transform 300ms, color 300ms;
```
Указывай какие свойства ты анимируешь